src/Raven.Server/ServerWide/Commands/UpdateValueForDatabaseCommand.cs

```diff
@@ -45,10 +50,12 @@ public virtual unsafe void Execute(ClusterOperationContext context, Table items,
                 itemKey = GetItemId();
             }
 
+            Debug.Assert(updatedValue.Action == Action.Update && updatedValue.Value != null);
+
             using (Slice.From(context.Allocator, itemKey, out Slice valueName))
             using (Slice.From(context.Allocator, itemKey.ToLowerInvariant(), out Slice valueNameLowered))
             {
-                ClusterStateMachine.UpdateValue(index, items, valueNameLowered, valueName, itemBlittable);
+                ClusterStateMachine.UpdateValue(index, items, valueNameLowered, valueName, updatedValue.Value);
```

Comment:
there are 0 methods that override `GetResult()`
1. We can remove it
2. It means probably that this value is not used anywhere so we can dispose it?
3. We can make UpdatedValue disposable and dispose it there?

Reply:
done